1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -1,4 +1,7 @@
 export const fizzbuzz = (value: number): string => {
   if (isNaN(value) || value < 1 || value > 100) return "";
+
+  if (value % 3 === 0) return "Fizz";
```

Comment:
In the demo that Khalil did, he suggested to refactor this type of code after we see the comparison 3 times, so he suggests to go in small steps, and then when we see repetition we abstract and remove duplication.